extension/src/lambda_service.rs

```diff
@@ -22,26 +33,33 @@ use crate::{
   messages::{WaiterRequest, WaiterResponse},
 };
 
-use tokio_rustls::client::TlsStream;
-
 // Define a Stream trait that both TlsStream and TcpStream implement
 pub trait Stream: AsyncRead + AsyncWrite + Send {}
 impl Stream for TlsStream<TcpStream> {}
 impl Stream for TcpStream {}
 
+pub type AppClient = Client<HttpConnector, StreamBody<Receiver<Result<Frame<Bytes>>>>>;
+
 #[derive(Clone)]
-pub struct LambdaService {
+pub struct LambdaService<'a> {
   options: Options,
   initialized: Arc<AtomicBool>,
   healthcheck_url: Uri,
+  app_client: &'a AppClient,
```

Comment:
Since `Client`s are cheap to clone, perhaps could avoid proliferating a lifetime by letting the LambdaService own the `AppClient` and cloning if needed. The same underlying connection pool will be used in each clone.

Reply:
https://github.com/pwrdrvr/lambda-dispatch/pull/194

Reply:
Yeah, I had seen the clone and a comment that they were cheap to clone and was wondering about that.